src/browser/screens/choosePath/ChoosePathScreen.js

```diff
@@ -50,8 +50,7 @@ export default class ThirdScreen extends Component {
               <li key={role.id}
                   onClick={() => this.handleChooseRole(role)}>
                 <p>
-                  <strong>{role.name.split(' ')[0]}</strong>
-                  {role.name.split(' ')[1]}
+                  <strong>{role.name.split(' ')[0]}</strong> {role.name.split(' ')[1]}
```

Comment:
Fragile design - you assume that the first word will be "the one". So "Mobile warrior" will work but not "Paladin of Mobile"